Add ODSC and DevOps Days India (#151)
nikhita/tech-conferences-india: README.md

| Conference | Date | Venue | Description | Scholarship |
|------------|------|-------|-------------|-------------|

> | [Functional Conf](https://functionalconf.com/) |  Nov 13—17, 2019 |  ITC WelcomHotel, Bengaluru  | Functional Conf is designed to bring together the growing community of functional programmers under one roof.s | No |

Reviewer: ```suggestion
| [Functional Conf](https://functionalconf.com/) |  13-17 Nov |  ITC WelcomHotel, Bengaluru  | Functional Conf is designed to bring together the growing community of functional programmers under one roof. | No |
```

| [Open Source India](https://www.opensourceindia.in/) |  17-18 Oct |  NIMHANS Convention Centre, Bengaluru  | A 2-day conference with several workshops, primarily focusing on the trending technologies in Open Source | No |
| [PyCon India](https://in.pycon.org/2019/) |  12-15 Oct |  Chennai Trade Center and IIT Madras Research Park  | India's premier conference on using and developing the Python programming language | No |
| [JSFoo Bangalore](https://hasgeek.com/jsfoo/2019/) | 27-28 Sep | NIMHANS Convention Centre, Bangalore | JSFoo is in its ninth edition this year. The 2019 edition is a two-track event with talks and Birds of Feather (BOF) sessions | No |
| [React India](https://www.reactindia.io/) | 26-28 Sep | Goa | A 3 day conference with first day focussing on workshops & the next 2 days talks on things around React, React Native, GraphQL, VR & AR, Redux, Preact and more. | [Yes](https://www.notion.so/Scholarship-Program-React-India-2019-490794dd0a464db0a43b4c8c5440839f) |

> | [DevOps Days India 2019](https://devopsdaysindia.org/) | 13th - 14th Sept 2019 | Radisson Blue, Bengaluru | Global event complementing other Devopsdays events across the world| No |

Reviewer: ```suggestion
| [DevOps Days India](https://devopsdaysindia.org/) | 13-14 Sep | Radisson Blue, Bengaluru | Global event complementing other Devopsdays events across the world | [Yes](https://docs.google.com/forms/d/e/1FAIpQLSeYWn5xcuEenAIwhtuNUCU_2n7zZCx3bHUdMU8tbzUzBkNRrQ/viewform) |
```

| [DevOps India Summit](https://devopsindiasummit.com/) | 31 Aug-1 Sep | Marriott Hotel, Whitefield, Bengaluru |  2-day conference focusing on thoughts around the collaboration of business success; the collaboration of DevOps and Cloud. | Yes |
| [JSFoo: VueDay](https://hasgeek.com/jsfoo/2019-vueday/proposals) | 30 Aug | Bangalore International Centre | A day spent discussing Vue.js, developments in the Vue ecosystem and component architecture | Yes, for speakers |
| [WordCamp Nagpur](https://2019.nagpur.wordcamp.org) | 11 Aug | MIA Centre for Sports & Recreation, Nagpur | Primarily a Wordpress Conference, covering everything related to WordPress and include things such as blogging, coding, branding, and security. | Yes, for speakers |

> | [Open Data Science Conference](https://india.odsc.com/) | August 7th – 10th | Sheraton Grand Bangalore Hotel, Bangalore | Specialized data science event, with a focus on impactful tools and leading industry practices| Not Sure - Global ODSC does has scholarship - https://odsc.com/scholarship/ |

Reviewer: ```suggestion
| [Open Data Science Conference](https://india.odsc.com/) | 7-10 Aug | Sheraton Grand Bangalore Hotel, Bangalore | Specialized data science event, with a focus on impactful tools and leading industry practices | No |
```
